lib/tools.cpp

```diff
@@ -43,36 +43,58 @@ QString get_kb_layout_dir()
 #endif // BUNDLE_KEYBOARDLAYOUTS
 }
 
-/*! Helper function to get possible location of layout files.
+/*! Helper function to add custom location of color schemes.
+*/
+QStringList custom_color_schemes_dirs;
```

Comment:
Would it be feasible to use `static` or anonymous namespace to avoid exporting this symbol?